lib/grid_printer.rb

```diff
@@ -0,0 +1,29 @@
+class GridPrinter
+  attr_reader :stdout, :grid
+
+  def initialize(stdout, grid)
+    @stdout = stdout
+    @grid = grid
+  end
+
+  def print_grid
+    stdout.puts(replace_template_with_value_from_grid)
+  end
+
+  def replace_template_with_value_from_grid
+   grid_template_model_mustache.gsub(/[A-C][1-3]/) { |cell| grid.get_value(cell) }
+  end
+
+  def grid_template_model_mustache
```

Comment:
I wouldn't call this mustache since it isn't really mustache. It's just a grid template!

Reply:
LOL oops 😆